modules/distr-tracing-tempo-monolithic-config-receiver-tls.adoc

```diff
@@ -0,0 +1,73 @@
+:_mod-docs-content-type: REFERENCE
+[id="distr-tracing-tempo-config-receiver-tls_{context}"]
+= Configure receiver TLS
+
+The `TempoMonolithic` custom resource (CR) allow to configure TLS on the distributor receivers when is not used with the gateway. You can specify a certificate, first you need to create a Secret with the tls.key and tls.crt keys, Those keys will contains the certificate and the key. Optionally you can specify the CA in a ConfigMap if you want to enable mutual TLS authentication.
+
+The following is an example of `TempoStack` CR with the TLS enable on receivers:
+
+
+.Sample Tempo CR with TLS enable on receivers
+[source,yaml]
+----
+apiVersion: tempo.grafana.com/v1alpha1
+kind:  TempoStack
+metadata:
+  name: simplest
+  namespace: chainsaw-tls-receivers
+spec:
+  storage:
+    secret:
+      name: minio
+      type: s3
+  storageSize: 1Gi
+  resources:
+    total:
+      limits:
+        memory: 2Gi
+        cpu: 2000m
+  template:
+    distributor:
+      tls:
+        enabled: true <1>
+        certName: my-tls-secret <2>
+        caName: my-ca <3>
+    queryFrontend:
+      jaegerQuery:
+        enabled: true
+----
+
+<1> Must be set to `true` for enable TLS on the distributor
+<2> The secret that contains the tls.key and tls.crt files.
+<3> Optionally a CA if you want to enable mTLS
+
+In OpenShift you can also use the Service serving certificates, if you want to use it you only need to enabled TLS, no need to specify certificate secret. If you specify one, that will be use, otherwise OpenShift service certificates are used, this mode doesn't support mTLS at the moment.
```

Comment:
By "Service serving certificates", do you mean
this
https://docs.openshift.com/container-platform/4.14/security/certificate_types_descriptions/service-ca-certificates.html#cert-types-service-ca-certificates
and also this
https://docs.openshift.com/container-platform/4.14/security/certificates/service-serving-certificate.html
?

Reply:
Yes,